ORStools/gui/ORStoolsDialog.py

```diff
@@ -433,6 +438,32 @@ def __init__(self, iface, parent=None):
         self.routing_fromline_list.model().rowsMoved.connect(self._reindex_list_items)
         self.routing_fromline_list.model().rowsRemoved.connect(self._reindex_list_items)
 
+    def _save_vertices_to_layer(self):
+        """Saves the vertices list to a temp layer"""
+        items = [
+            self.routing_fromline_list.item(x).text()
+            for x in range(self.routing_fromline_list.count())
+        ]
+
+        if len(items) > 0:
+            crs = QgsCoordinateReferenceSystem("EPSG:4326")
+            fields = QgsFields()
+            fields.append(QgsField("ID", QVariant.Int))
+            point_layer = QgsVectorLayer("Point?crs=4326", "Vertices", "memory")
```

Comment:
This is missing the `EPSG:` in the URI, which is why it's asking for a CRS.
Also, the URI can be used to set the ID field, compare the sample usage here: https://qgis.org/pyqgis/3.16/core/QgsVectorLayer.html#qgis.core.QgsVectorLayer

Reply:
Done